PYME/LMVis/Extras/objectMeasurements.py

```diff
@@ -85,11 +85,23 @@ def OnMeasure(self, event):
 
         chans = pipeline.colourFilter.getColourChans()
 
-        ids = set(pipeline.mapping['objectID'].astype('i'))
+        # If we're not using objectIDs from an image, look for other clustering labels
```

Comment:
```suggestion
        # If we're not using objectIDs from an image, look for other clustering labels
        # TODO - rather than trying a few pre-set objectID alternatives, make this a dialog instead
        # - look for objectID and carry on if present
        # - if not present, display a dialog "No objectID found - did you segment objects? Either cancel, 
        # segment, and come back, or choose an alternative column to use as an ID.
```